app/src/main/res/layout/activity_home.xml

```diff
@@ -2,8 +2,99 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/busify"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".ui.home.HomeActivity">
+    android:background="#FFFAE8"
```

Comment:
remove background color here we'll try to stick to figma design

Reply:
I remove the background color